docs/next/guides/building-and-testing/building.md

```diff
@@ -9,83 +9,263 @@ tags:
+| `./wrappers/vue/package.json`     | The Vue package                                     |
 
-Each of the wrappers has its own framework-specific build environment and scripts. The build process can be triggered by either navigating the the wrapper directory (e.g. `./wrappers/react-handsontable/` and running:
+## Running your first build
 
-*   `npm run build`
+To run your first build:
+1. Install [Node.js](https://nodejs.org/) (version **15.11**+).
+2. Install [npm](https://www.npmjs.com/) (version **7.17**+).
+3. Clone the [Handsontable repository](https://github.com/handsontable/handsontable).
+4. From the root directory, run `npm install`.<br>All the required dependencies get installed.
+5. From the root directory, run `npm run build`.<br>All the Handsontable packages get built.
 
-Or by utilizing one of the helper scripts in the root directory:
+## Building the packages
 
-*   `npm run in` - Runs a command for the specified project. For Example:  
-    `npm run in angular build` - runs the `build` command for the `@handsontable/angular` project.  
-    Shorthands are also available - the aforementioned command works as `npm run in angular build` as well.
-*   `npm run all` - Runs a command for all the packages. For example:  
-    `npm run all build` will run the `build` command for all the defined packages.  
-    The order in which the packages are executed is defined in the `./scripts/run-all.mjs` script.
+You can either build all the packages at once, or build each package individually.
 
-## Running your first build
+### Building all the packages
+
+To build all the packages at once:
+1. Make sure you meet the [building requirements](#building-requirements).
+2. Go to the root directory.
+3. Run `npm run build`.<br>The script builds the following packages:
+     - The JavaScript package
+     - The Angular package
+     - The React package
+     - The Vue package
+     - A code examples package
+
+#### Root directory build tasks
+
+You can also run individual root directory `build` tasks:
+
+::: details Root directory build tasks
+`build:codesandbox`
+  - PLACEHOLDER
+:::
+
+### Building the JavaScript package
+
+To build the JavaScript package:
+1. Make sure you meet the [building requirements](#building-requirements).
+2. Go to `./handsontable`.
+3. Run `npm run build`.<br>Only the JavaScript package builds.
+
+To build the JavaScript package from the root directory:
+1. Make sure you meet the [building requirements](#building-requirements).
+2. Go to the root directory.
+3. Run `npm run in handsontable build`.<br>Only the JavaScript package builds.
+
+#### JavaScript build tasks
+
+You can also run individual JavaScript `build` tasks:
+
+::: details JavaScript build tasks
+`npm run build:commonjs`
+  - Transpiles files into the `CommonJS` format.
+  - During release, these files are published into the npm repository.
+
+`npm run build:es`
+  - Transpiles files into the `import/export` format.
```

Comment:
`ES` or `ESM` format sounds more familiar to me, but maybe it's just me 😆

Reply:
Done: 8841e1e